<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMirrorExtend</class>
 <widget class="QDialog" name="DialogMirrorExtend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mirror Extend</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>px</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QSpinBox" name="spinBoxExtendPixels">
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogMirrorExtend</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogMirrorExtend</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
